tools/SourceKit/tools/sourcekitd/lib/API/Requests.cpp

```diff
@@ -1824,10 +1840,6 @@ static void reportCursorInfo(const RequestResult<CursorInfoData> &Result,
       Receiver.set(KeyUSR, USR);
     }
   }
-  if (Info.IsSystem)
-    Elem.setBool(KeyIsSystem, true);
-  if (!Info.TypeInterface.empty())
```

Comment:
As far as I can tell, this was always empty (and no tests check it). Though I just noticed that I didn't remove the print in the test. Should I also remove the key in utils/gyb_sourcekit_support/UIDs.py?

Reply:
Go for it. It looks like it has been dead code since 2016 (8f1c239d01cc97c4696835dc7d11bc8f5a9d89b4).